src/functional_test_runner/lib/config/schema.js

```diff
@@ -88,6 +88,10 @@ export const schema = Joi.object().keys({
     elasticsearch: urlPartsSchema(),
   }).default(),
 
+  elasticsearchServerArgs: Joi.array(),
+  kibanaServerArgs: Joi.array(),
+
+  // env allows generic data, e.g. for fields unrelated to URLs
```

Comment:
__nit:__ since we didn't manage to figure out what this description means, let's just get rid of it or write a better one :)

Reply:
I don't think `env` should be used at all, and we should remove it. It encourages sloppy configs, because anything can go in it. I'm not going to remove it here, but it doesn't feel like it belongs at all.